GCP Image Builder (#1270)
zenml-io/zenml: docs/book/component-gallery/image-builders/gcloud-build.md

---
description: How to build container images with Google Cloud Build
---

The Google Cloud image builder is an [image builder](./image-builders.md) flavor provided
with the ZenML `gcp` integration that uses [Google Cloud Build](https://cloud.google.com/build)
to build container images.

## When to use it

You should use the Google Cloud image builder if:
* you're **unable** to install or use [Docker](https://www.docker.com) on your client machine.
* you're already using GCP.

> * your stack is conformed mainly by other Google Cloud components such as the [GCS Artifact Store](../artifact-stores/gcloud-gcs.md) or the [Vertex Orchestrator](../orchestrators/gcloud-vertexai.md).

Reviewer: ```suggestion
* your stack is mainly composed of other Google Cloud components such as the [GCS Artifact Store](../artifact-stores/gcloud-gcs.md) or the [Vertex Orchestrator](../orchestrators/gcloud-vertexai.md).
```

## How to deploy it

> In order to use the ZenML Google Cloud image builder is enabling Google Cloud Build relevant APIs on the Google Cloud project.

Reviewer: ```suggestion
In order to use the ZenML Google Cloud image builder you need to enable Google Cloud Build relevant APIs on the Google Cloud project.
```
